src/utils/common.ts

```diff
@@ -681,3 +684,35 @@ export function decryptedComputed(
       set: setEncryptedState,
     });
 }
+
+interface DirectionParams {
+  innerTx?: ITx;
+  outerTx?: ITx;
+  transactionCustomOwner?: AccountAddress;
+  transactionOwner?: AccountAddress;
+  ownershipStatus?: ObjectValues<typeof AE_TRANSACTION_OWNERSHIP_STATUS>;
+  txOwnerAddress?: string;
+  activeAccountAddress?: string;
+}
+export function getDirection({
```

Comment:
Please add empty line before this one.

Reply:
Seems like I can just use the `getTxDirection` directly, I'm not sure if this extra condition is even necessary, seems to work fine without it.

if ((tx as ITx)?.function === TX_FUNCTIONS.claim) {
return TX_DIRECTION.received;
}